recipes-bsp/optee-os/optee-os-tadevkit_3.19.0.bb

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
You should upstream the optee-os-tadevkit to ISAR instead of patching it here.

Reply:
I am still struggling of finding a best solution for it...

optee-os depends on optee-ftpm, the later depends optee-os-tadevkit which is also produced by optee-os building...

I am afraid if I put the optee-os binary and the optee-os-tadevkit into a some debian/control file, it would cause looping dependency...

Reply:
Is there a way to split the optee-os build so that the final assembly of core and TAs can be done in a separate recipe? That way, you could build optee-os + tadevkit in one run, then the TAs, and finally assemble the firmware image.

Reply:
Good idea, however it requires to modify the current build mechanism of the optee-os, since currently there is no ways for early TAs to be integrated in a later stage... not a small one.

Reply:
First version of upstreaming the tadevkit:
https://groups.google.com/g/isar-users/c/u03AQp6B1eM